react/tic-tac-toe/src/app/components/Input/styles.module.scss

```diff
@@ -0,0 +1,87 @@
+@import 'src/scss/colors';
+@import 'src/scss/fonts';
+@import 'src/scss/variables';
+$input-border: $gainsboro-gray;
+$focus-color: $wolox-blue;
+$red-error: $red;
+
+.container {
+  margin-bottom: 30px;
+  position: relative;
+  width: 100%;
+
+  &::after {
+    border: 1px solid $input-border;
+    border-radius: 0.25rem;
+    bottom: 0;
+    content: '';
+    left: 0;
+    margin: 0;
+    padding: 0;
+    pointer-events: none;
+    position: absolute;
+    right: 0;
+    top: 0;
+    transition: border ;
+    transition-duration: $default-transition-duration * 2;
+    transition-timing-function: $border-transition-timing;
+  }
+
+  &.focused {
+    &:after {
+      border: 2px solid $focus-color;
+    }
+
+    .label {
+      color: $focus-color;
+    }
+  }
+
+  &.focused .label,
+  &.has-value .label {
+    font-size: $text-small;
```

Comment:
Nice, but I suggest that since it's just a visual change, you do this animation with `transform: scale()` since it will [perform better](https://medium.com/outsystems-experts/how-to-achieve-60-fps-animations-with-css3-db7b98610108) on mobile devices.